ui/src/app.css

```
@@ -1,6 +1,13 @@
 /* IBM Plex Mono for monospace text */
 @import url('https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:ital,wght@0,400;0,500;0,600;1,400&display=swap');
 
+/* UIUA array language font */
+@font-face {
+  font-family: 'Uiua';
```

Stylelint (17.3.0)

[error] Unexpected quotes around "Uiua" (font-family-name-quotes)

(font-family-name-quotes)